README.md

```diff
@@ -50,7 +50,7 @@
             - [:include](#include)
             - [:use](#use)
     - [Spinner](#spinner)
-    - [inline-documentation](#inline-documentation)
+    - [rust docs in org mode](#rust-docs-in-org-mode)
```

Comment:
@samhedin I just realized that's not what "inline" means. But unfortunately I can't think of a better section name. I just want to avoid confusions.

Reply:
Agreed!